NEWS.md

```diff
@@ -1,76 +1,113 @@
-3.19.1
+3.20.1
 ======
 
 ## Features
 
- * HTTP load balancer (#2347)
- * Slack destination (#2451)
- * Add Cisco Catalyst formatted triplets support to cisco-parser() (#2394)
- * Add RFC5424 syslog support to the system() source (FreeBSD 12.0 support) (#2430)
- * Add network `interface()` option to network sources (#2389)
- * Add so-reuseport() to network drivers (#2379)
- * Enable supporting HTTP redirects (#2136)
+ * Add persist-tool (#2511)
+ * Collectd destination (#1433)
+ * Config reload status feedback (#2367)
+ * Netskope parser (#2541)
+ * Websense parser (#2471)
+ * Json list support (#2536)
+ * Python-destination: send can return worker_insert_result_t, flush support (#2487)
+ * Python: internal() source exposed via syslogng.Logger (#2505)
+ * Xml/geoip2: make prefix optional (#2538)
+ * Xml-parser: add list support (#2544)
 
 ## Bugfixes
 
- * Fix frequent disconnects of syslog() with TLS (#2432)
- * Fix possible refcount leak during reload/shutdown (#2434)
- * Fix message storm on trace level (#2425)
- * Fix use after free in file destinations (time-reap) (#2418)
- * Fixing a few memleaks in the Java destination (#2417)
- * Fix telegram dst default ca dir (#2416)
- * Fix prefix handling in $(list-concat) and $(strip) (#2405)
- * Fixing an eventfd leak with ivykis<=0.38 (threaded destinations) (#2404)
- * Process flush result after worker thread exits (threaded destinations) (#2402)
- * hdfs: do not try to write unopened file (#2391)
- * Fix leaks in redis() destination (#2383)
- * Block location tracking fixes (#2378)
- * Fix $(basename) and $(dirname) in the presence of a prefix (#2371)
- * Fixing a false positive corruption detection in non-reliable diskq (#2356)
- * Check if /proc/kmsg can be opened in system-source (#2408)
- * Fix include guard in systemd-journal (#2445)
- * Remove hexadecimal and octal number parsing from templates (#2401)
+ * Cfg version (#2548)
+ * DNS memory leak/segfault fix (#2500)
+ * Elasticsearch2: fix bulk send for client-mode("http") (#2478)
+ * Few leak fix (#2563)
+ * Filter-re: ref/unref NVTable around regex eval (#2494)
+ * Fix app-parser() per reload memory leak (#2469)
+ * Fix non-reliable disk-queue truncating problem on load (#2406)
+ * Fix threaded source/destination crash when reverting configuration (#2555)
+ * Http: add missing free for self->body_template (#2558)
+ * Java, elasticsearch2, explicit unsupport for named templates (#2534)
+ * Loggen: parameter handling (#2477)
+ * Persist-tool: remove criterion as dependency (#2547)
```

Comment:
we will release persist-tool in this release, I would consider this as an internal patch during development.